### DISCUSSION TITLE
SP-API scheduling of vendor central reports through SQS

### DISCUSSION THREAD

**Discussion by voxum** [amzn/selling-partner-api-models#3445] [General]
I've been successful in getting multiple seller accounts reporting scheduled and received notifications through our SQS queue, but we are having issues scheduling some of the vendor reports needed. I've noticed in the docs that some reports require certain role permissions and we've made sure to select them all, but with no success. Is there any way to schedule and receive notifications through SQS or event bridge for GET_VENDOR_SALES_REPORT ?

**Comment by sungolivia** [comment]
Have you selected them through developer profile? After you have gotten the roles approved, you need to select them again for your application. Just making sure!
https://developer-docs.amazon.com/sp-api-blog/docs/choosing-roles-for-developer-profile-and-sp-api-app

**Comment by MeenaAmz** [comment]
REPORT_PROCESSING_FINISHED notification is not available yet for vendor reports.

**Reply by NA-Dev** [reply]
@MeenaAmz why are SO MANY things in the API not available to Vendors? We should be your primary customers, but third party sellers are getting all the tools. I was excited today to start using EventBridge and SQS Queues, but it is not very useful for us since everything is sellers only.

**Comment by MeenaAmz** [comment]
@NA-Dev [LISTINGS_ITEM_ISSUES_CHANGE](https://developer-docs.amazon.com/sp-api/docs/notifications-api-v1-use-case-guide#listings_item_issues_change) and [PRODUCT_TYPE_DEFINITIONS_CHANGE](https://developer-docs.amazon.com/sp-api/docs/notifications-api-v1-use-case-guide#product_type_definitions_change) notifications are available for vendors. Thanks for your feedback. We have submitted your feature request to the appropriate team.

**Reply by NA-Dev** [reply]
@MeenaAmz Yes please, we first need BRANDED_ITEM_CONTENT_CHANGE since Amazon lets any random third party edit PDPs without approval from the actual vendor.
